src/scripts/main.js

```diff
@@ -355,6 +355,55 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people); // you can remove it
 
 // write your code here
+
+const table = document.querySelector('.dashboard');
+
+for (const person of people) {
+  const trElement = document.createElement('tr');
+
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
+
+  let sex = '';
+
+  if (person.sex === 'm') {
+    sex = 'Male';
+  } else {
+    sex = 'Female';
+  }
+
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = person.name;
+  trElement.appendChild(nameCell);
```

Comment:
you do the same 6 times - create a function that accepts correct cell value (text content) as an argument and does the same
Reuse this function